frontend/javascripts/dashboard/dashboard_view.tsx

```diff
@@ -281,20 +289,26 @@ class DashboardView extends PureComponent<PropsWithRouter, State> {
       !activeUser.novelUserExperienceInfos.hasSeenDashboardWelcomeBanner ? (
         <WhatsNextHeader activeUser={activeUser} onDismiss={this.onDismissWelcomeBanner} />
       ) : null;
+    this.state.pricingPlanStatus?.isAlmostExceeded;
 
+    // ToDo enable components below once pricing goes live
     const pricingPlanWarnings =
-      user.isAdmin && this.state.organization && this.state.storageSpaceInfo ? (
-        <PlanAboutToExceedWarning
-          organization={this.state.organization}
-          usedStorageSpace={this.state.storageSpaceInfo.usedStorageSpace}
-        />
+      this.state.organization &&
+      this.state.pricingPlanStatus?.isAlmostExceeded &&
+      !this.state.pricingPlanStatus.isExceeded ? (
+        <PlanAboutToExceedAlert organization={this.state.organization} />
+      ) : null;
+    const pricingPlanErrors =
+      this.state.organization && this.state.pricingPlanStatus?.isExceeded ? (
+        <PlanExceededAlert organization={this.state.organization} />
       ) : null;
 
     return (
       <NmlUploadZoneContainer onImport={this.uploadNmls} isUpdateAllowed>
         {whatsNextBanner}
-        <div className="container">
-          {pricingPlanWarnings}
+        <div className="container propagate-flex-height" style={{ minHeight: "66vh" }}>
+          {/* {pricingPlanWarnings}
+          {pricingPlanErrors} */}
```

Comment:
remove?

Reply:
Mhmm, the components are intentionally commented out/disabled. We will need to re-enable them once the pricing goes live early next year. This is what the comment in line 294 is referring to.